workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md

```diff
@@ -65,6 +65,37 @@ Complete the following tasks:
 
 ![](../.gitbook/assets/ex3/image004.png)
 
+* [ ] To complete the creation of the Cloudant service instance, accept the default value for region and keep the **Lite** price plan as the selected plan as shown in the following figure.
+
+![](../.gitbook/assets/ex3/image005.jpg)
+
+* [ ] Accept the default values for service name and resource group.
+* [ ] In **Available authentication methods**, select **Use only IAM**, as shown in the following figure. 
+
+![](../.gitbook/assets/ex3/image006.jpg)
+
+{% hint style="info" %}
+:memo: The Cloudant service has two authentication methods that are available: **Use both legacy credentials and IAM**, which means that both IAM and Legacy credentials can be used to access the account, and Use only IAM, which means that only IAM credentials are provided through service binding and credential generation. 
```

Comment:
This is a good memo. But I can't see the final form

Reply:
I can see the final form in VS COde